src/Symfony/Component/OptionsResolver/Debug/OptionsResolverIntrospector.php

```diff
@@ -101,4 +101,14 @@ public function getDeprecation(string $option): array
     {
         return ($this->get)('deprecated', $option, \sprintf('No deprecation was set for the "%s" option.', $option));
     }
+
+    /**
+     * @return \Closure[]
+     *
+     * @throws NoConfigurationException when no nested option is configured
+     */
+    public function getNestedOptions(string $option): array
```

Comment:
shouldn't you update the descriptor in the form component to use this new introspection method ?

Reply:
btw, I think this description (used for the `debug:form` command) forgot entirely the case of nested options.

Reply:
yea, I realized that while working on this PR, so that would be the next step: adding nested options output in the form command and its descriptors

Reply:
btw, that can be done in a separate PR

Reply:
See https://github.com/symfony/symfony/pull/59871